requirements.txt

```diff
@@ -22,4 +22,4 @@ scipy==1.2.1
 six==1.10.0
 torch==0.4.0
 torchvision==0.2.1
-
+tensorboard==2.1.0
```

Comment:
It is incompatible with torch 0.4.0. It is better to install both tensorboard and tensorboardX==2.0
Full requirements.txt:
```
asn1crypto==0.22.0
certifi==2017.4.17
cffi==1.10.0
decorator==4.4.0
idna==2.6
networkx==2.3
numpy==1.14.3
olefile==0.45.1
packaging==16.8
pandas==0.24.2
Pillow==4.2.1
pycosat==0.6.3
pycparser==2.18
pycrypto==2.6.1
pyOpenSSL==17.0.0
pyparsing==2.2.0
python-dateutil==2.8.0
pytz==2019.1
requests==2.21.0
scikit-learn==0.20.3
scipy==1.2.1
six==1.10.0
torch==0.4.0
torchvision==0.2.1
tensorboard==2.1.0
tensorboardX==2.0
```

Reply:
Done